src/ensembl/src/content/app/browser/drawer/drawerActions.ts

```diff
@@ -51,3 +51,25 @@ export const toggleDrawer: ActionCreator<
     })
   );
 };
+
+export const closeDrawer: ActionCreator<
+  ThunkAction<void, any, null, Action<string>>
+> = () => (dispatch, getState: () => RootState) => {
+  const activeGenomeId = getBrowserActiveGenomeId(getState());
+
+  if (!activeGenomeId) {
+    return;
+  }
+
+  dispatch(
+    toggleDrawerForGenome({
+      [activeGenomeId]: false
+    })
+  );
+
+  dispatch(
+    changeDrawerViewForGenome({
+      [activeGenomeId]: ''
+    })
```

Comment:
We can batch these two together?

https://react-redux.js.org/api/batch

Reply:
This is done now